lib/index.js

```diff
@@ -8,14 +8,51 @@
   The above copyright notice and this permission notice shall be
   included in all copies or substantial portions of this Source Code Form.
 */
+const assert = require('assert').strict;
+
 const Input = require('postcss/lib/input');
 
 const Parser = require('./ValuesParser');
+const SubInput = require('./SubInput');
 const { stringify } = require('./ValuesStringifier');
 
+const NEWLINE = '\n'.charCodeAt(0);
+const FEED = '\f'.charCodeAt(0);
+const CR = '\r'.charCodeAt(0);
+
+function positionAfter(node, chunks) {
+  let { line } = node.source.start;
+  let { column } = node.source.start;
+  for (const chunk of chunks) {
+    for (let i = 0; i < chunk.length; i++) {
+      const code = chunk.charCodeAt(i);
+      if (
+        code === NEWLINE ||
+        code === FEED ||
+        (code === CR && chunk.charCodeAt(i + 1) !== NEWLINE)
+      ) {
+        column = 1;
+        line += 1;
+      } else {
+        column += 1;
+      }
+    }
+  }
+
+  return { line, column };
+}
+
 module.exports = {
   parse(css, options) {
-    const input = new Input(css, options);
+    let input;
+    if (options.context) {
```

Comment:
`options.context` isn't a documented option (neither are `options.lineInContext` nor `columnInContext`) so I can't comment on them. documentation isn't only useful to end users after merge, it's a key component of new feature PR reviews.